src/NumSharp.Core/Math/NDArray.cumsum.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NumSharp
+{
+    public partial class NDArray
+    {
+        public NDArray cumsum()
+        {
+            return np.cumsum(this);
+        }
+    }
+
+    public static partial class np
+    {
+        public static NDArray cumsum(NDArray a)
```

Comment:
Add documentation similar to [this](https://github.com/SciSharp/NumSharp/blob/master/src/NumSharp.Core/Random/np.random.randint.cs#L14)